### ISSUE TITLE
ambassador 0.33 yaml to be corrected there was close braces on image end

### ISSUE THREAD

**Issue by satish-cevp** [datawire/ambassador#438]
**Describe the bug**

**To Reproduce**

**Expected behavior**

**Versions (please complete the following information):**
 - Ambassador: [e.g. 0.32.1]
 - Kubernetes environment [e.g. Minikube, bare metal, Google Kubernetes Engine]
 - Version [e.g. 1.8.1]

**Additional context**

**Comment by richarddli** [comment]
Thanks for the bug report. I believe you're referring to:

```
      - name: statsd
        image: quay.io/datawire/statsd:0.33.0)
      restartPolicy: Always
```

in the Makefile, with the extra `)`. We'll fix this ASAP.

**Comment by richarddli** [comment]
This is fixed in 0.33.1. Thanks for the bug report.
